feat: Set background colour of div that contains the EBD (off-white)

hf-kklein opened the pull request:
to the same as the SVG background

@OLILHR das sieth aber irgendwie mist aus :-( dann machen wir den SVG hintergrund in off-white sobald die farbe feststeht.

![{193702B0-DEEE-4017-AF65-779C8D3011B5}](https://github.com/user-attachments/assets/da9a151a-2527-44ba-8e50-6c7b83e985f8)

Commit:
feat: Set background colour of div that contains the EBD

to the same as the SVG background

OLILHR commented:
> @OLILHR das sieth aber irgendwie mist aus :-( dann machen wir den SVG hintergrund in off-white sobald die farbe feststeht.

was sieht nicht gut aus? Das blau mit der watermark?
ich finde das einfach zu unruhig; die watermark ist viel zu penetrant.

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/bc9b0ffe-c956-4667-9f75-cdf88c834c48" />

also entweder ja - irgendein off-white oder vielleicht eine schwachere Variante von dem "Blau"
<img width="1592" alt="image" src="https://github.com/user-attachments/assets/073a1db5-4bb2-4cea-a060-6f251466cdc8" />

weiß nicht was Annika sagt. Der Kontrast zwischen Watermark und Background muss auf jeden Fall kleiner werden.

emmastoeck commented:
Aus meiner Sicht ist das Wasserzeichen auch zu prominent in der Version. Das Problem ist dass das Auge nicht weiß, wo der Inhalt ist, der relevant ist (ebds) und, welcher Inhalt eigentlich irrelevant ist für den Benutzer (Wasserzeichen). Zusätzlich verschwindet unsere Fußleiste gänzlich.. An sich aber eine modernere Version als es off-white zu machen. Aber vielleicht mit einem helleren Blauton?

Commit:
fix: switch from pastell blau to off-white

to align with upcoming EBD background: https://github.com/Hochfrequenz/rebdhuhn/pull/337

Commit:
prettier
